analysers/Analyser_Osmosis.py

```diff
@@ -204,6 +204,34 @@ class Analyser_Osmosis(Analyser):
 CREATE INDEX idx_multipolygons_poly_proj ON {0}.multipolygons USING gist(poly_proj);
 CREATE INDEX idx_multipolygons_tags ON {0}.multipolygons USING gist (tags);
 ANALYZE {0}.multipolygons;
+"""
+
+    sql_create_polygons = """
+CREATE UNLOGGED TABLE {0}.polygons AS
+SELECT
+    'R' AS type,
+    *
+FROM
+    {0}.multipolygons
+UNION ALL
+SELECT
+    'W' AS type,
+    id,
+    tags,
+    CASE WHEN ST_IsValid(linestring) = 't' AND ST_IsSimple(linestring) = 't' AND ST_IsValid(ST_MakePolygon(ST_Transform(linestring, {1}))) THEN ST_MakePolygon(linestring) ELSE NULL END AS poly,
+    CASE WHEN ST_IsValid(linestring) = 't' AND ST_IsSimple(linestring) = 't' AND ST_IsValid(ST_MakePolygon(ST_Transform(linestring, {1}))) THEN ST_MakePolygon(ST_Transform(linestring, {1})) ELSE NULL END AS poly_proj,
+    true AS is_valid
```

Comment:
More a question than feedback, but: 
if the way-polygons are always valid, why not simply restrict the multipolygon-polygons to valid ones, and drop the is_valid output here?

Also this is a bit confusing because is_valid may be true, while `poly_proj` can be `NULL` due to being invalid (in rare cases)